### PR TITLE
explain more things

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

File: Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst [hunk 1]
```diff
@@ -81,10 +81,21 @@ In this case, the file *Index.html* will be loaded. Here you see an extract of t
...
 
-* `<f:format.nl2br>[…]</f:format.nl2br>` modifies linebreaks (new lines) to `<br />` tags.
-* `<f:link.action action="edit">` creates a link tag that links to the :php:`editAction()` of the current controller.
-* `<f:for each="{paginator.paginatedItems}" as="blog">[...]</f:for>` iterates over the paginated Blog objects found in Blogs.
-
+* `<f:format.nl2br>[…]</f:format.nl2br>` :
+   modifies linebreaks (new lines) to `<br />` tags.
+* `<f:link.action action="edit">` :
+  creates a link tag that links to the :php:`editAction()` of the current controller.
+* `<f:for each="{paginator.paginatedItems}" as="blog">[...]</f:for>` :
```
Comment (mbrodala) [inline Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst:88]:
```suggestion
* `<f:for each="{paginator.paginatedItems}" as="blog">[…]</f:for>` :
```

File: Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst [hunk 2]
```diff
@@ -81,10 +81,21 @@ In this case, the file *Index.html* will be loaded. Here you see an extract of t
...
+   iterates over the paginated Blog objects found in Blogs.
+* `<f:link.action action="index" controller="Post" arguments="{blog : blog}">{blog.title} ({f:translate(key: 'blog.numberOfPosts', arguments: '{numberOfPosts: \'{blog.posts -> f:count()}\'}')})</f:link.action>` :
```
Comment (mbrodala) [inline Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst:90]:
```suggestion
* `<f:link.action action="index" controller="Post" arguments="{blog : blog}">[…]</f:link.action>` :
```

File: Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst [hunk 3]
```diff
@@ -81,10 +81,21 @@ In this case, the file *Index.html* will be loaded. Here you see an extract of t
...
+   creates a link to the `indexAction` method of the `PostController` which is `public function indexAction(Blog $blog, $tag = null)`.
```
Comment (mbrodala) [inline Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst:91]:
```suggestion
   creates a link to the :php:`indexAction` method of the :php:`PostController` which is :php:`public function indexAction(Blog $blog, $tag = null)`. 
```

File: Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst [hunk 4]
```diff
@@ -81,10 +81,21 @@ In this case, the file *Index.html* will be loaded. Here you see an extract of t
...
+   modifies linebreaks (new lines) to `<br />` tags.
+* `<f:link.action action="edit">` :
+  creates a link tag that links to the :php:`editAction()` of the current controller.
+* `<f:for each="{paginator.paginatedItems}" as="blog">[...]</f:for>` :
+   iterates over the paginated Blog objects found in Blogs.
+* `<f:link.action action="index" controller="Post" arguments="{blog : blog}">{blog.title} ({f:translate(key: 'blog.numberOfPosts', arguments: '{numberOfPosts: \'{blog.posts -> f:count()}\'}')})</f:link.action>` :
+   creates a link to the `indexAction` method of the `PostController` which is `public function indexAction(Blog $blog, $tag = null)`. 
+   The first parameter blog is passed to this method. A temporary variable `numberOfPosts` is filled by the `f:count()` method.
+   The count method gives the numbers of posts. The text key `blog.numberOfPosts`of the file 
+   :file:`Resources/Private/Language/locallang.xml` refers to the text `%d posts`.
```
Comment (mbrodala) [inline Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst:94]:
```suggestion
   The blog stored in the variable `{blog}` is passed as `blog` parameter to the action.

* `{f:translate(key: 'blog.numberOfPosts', arguments: '{numberOfPosts: \'{blog.posts -> f:count()}\'}')}`
   The translation key `blog.numberOfPosts` refers to the translation file:`Resources/Private/Language/locallang.xml`.
   The number of `{blog.posts}` is counted using the `f:count()` viewhelper and passed as argument to the `f:translate()` viewhelper. This viewhelper inserts the argument into the translation text `%d posts`. 
```

File: Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst [hunk 5]
```diff
@@ -81,10 +81,21 @@ In this case, the file *Index.html* will be loaded. Here you see an extract of t
...
+* `<f:link.action action="edit">` :
+  creates a link tag that links to the :php:`editAction()` of the current controller.
+* `<f:for each="{paginator.paginatedItems}" as="blog">[...]</f:for>` :
+   iterates over the paginated Blog objects found in Blogs.
+* `<f:link.action action="index" controller="Post" arguments="{blog : blog}">{blog.title} ({f:translate(key: 'blog.numberOfPosts', arguments: '{numberOfPosts: \'{blog.posts -> f:count()}\'}')})</f:link.action>` :
+   creates a link to the `indexAction` method of the `PostController` which is `public function indexAction(Blog $blog, $tag = null)`. 
+   The first parameter blog is passed to this method. A temporary variable `numberOfPosts` is filled by the `f:count()` method.
+   The count method gives the numbers of posts. The text key `blog.numberOfPosts`of the file 
+   :file:`Resources/Private/Language/locallang.xml` refers to the text `%d posts`. 
+   The fluid method `f:translate(...)` fetches this string and replaces its `%d` argument by the number of posts for the current blog.
```
Comment (mbrodala) [inline Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst:95]:
```suggestion
```

File: Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst [hunk 6]
```diff
@@ -81,10 +81,21 @@ In this case, the file *Index.html* will be loaded. Here you see an extract of t
...
+   `{blog.posts}` is the array of posts which belong to the current blog. The class `Blog` contains the protected variable `$posts`.
```
Comment (mbrodala) [inline Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst:96]:
```suggestion
   `{blog.posts}` is the list of posts which belong to the current blog. The class `Blog` contains a getter method `getPosts()` which is automatically used to access the list of posts (:php:`\TYPO3\CMS\Extbase\Persistence\ObjectStorage`).
```

File: Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst [hunk 7]
```diff
@@ -81,10 +81,21 @@ In this case, the file *Index.html* will be loaded. Here you see an extract of t
...
+   This is the member variable (of type `\TYPO3\CMS\Extbase\Persistence\ObjectStorage`) where the posts are stored, e.g. in `function addPost(Post $post)` .
```
Comment (mbrodala) [inline Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst:97]:
```suggestion
```

**Review by mbrodala — CHANGES_REQUESTED** [review]
Notice in general that Fluid does not have "methods", only viewhelpers.

File: Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst (hunk 2 above)
Comment (franzholz) [reply Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst:90]: All those parts are necessary for the following explanations.

Comment (mbrodala) [reply Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst:90]: I know, that's why I've split this into 2 bullets.

**Review by linawolf — APPROVED** [review]

File: Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst [hunk 8]
```diff
@@ -81,10 +81,23 @@ In this case, the file *Index.html* will be loaded. Here you see an extract of t
...
+*  `<f:link.action action="edit">` :
+   creates a link tag that links to the :php:`editAction()` of the current controller.
+*  `<f:for each="{paginator.paginatedItems}" as="blog">[…]</f:for>` :
+   iterates over the paginated Blog objects found in Blogs.
+*  `<f:link.action action="index" controller="Post" arguments="{blog : blog}">[…]</f:link.action>` :
+   creates a link to the :php:`indexAction` method of the :php:`PostController` which is :php:`public function indexAction(Blog $blog, $tag = null)`. 
+   The blog stored in the variable `{blog}` is passed as `blog` parameter to the action.
+*  `{f:translate(key: 'blog.numberOfPosts', arguments: '{numberOfPosts: \'{blog.posts -> f:count()}\'}')}`
+   The translation key `blog.numberOfPosts` refers to the translation 
+   file:`Resources/Private/Language/locallang.xml`. The number of `{blog.posts}` is counted using the
```
Comment (mbrodala) [inline Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst:95]:
```suggestion
   file:`Resources/Private/Language/locallang.xlf`. The number of `{blog.posts}` is counted using the 
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

**Review by mbrodala — APPROVED** [review]

**Pull request merged**